src/System.Collections/src/System.Collections.csproj

```diff
@@ -3,29 +3,34 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <ProjectGuid>{D5FF747F-7A0B-9003-885A-FE9A63E755E5}</ProjectGuid>
-    <IsPartialFacadeAssembly Condition="'$(TargetGroup)' != 'netcore50aot'">true</IsPartialFacadeAssembly>
+    <IsPartialFacadeAssembly Condition="'$(TargetGroup)' != 'netcore50aot' OR '$(TargetGroup)' != 'net463'">true</IsPartialFacadeAssembly>
     <AssemblyName>System.Collections</AssemblyName>
-    <AssemblyVersion>4.0.12.0</AssemblyVersion>
+    <AssemblyVersion>4.1.0.0</AssemblyVersion>
+    <AssemblyVersion Condition="'$(TargetGroup)'=='netstandard1.3'">4.0.12.0</AssemblyVersion>
```

Comment:
yup that's the idea. Most of the changes that were required in all of these csprojs to be able to cross compile should go away.